python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_condition.py

```diff
@@ -649,6 +650,36 @@ def cron_tick_passed(
         )
         return CronTickPassedCondition(cron_schedule=cron_schedule, cron_timezone=cron_timezone)
 
+    @public
+    @staticmethod
+    def freshness_passed() -> "BuiltinAutomationCondition[AssetKey]":
+        """Returns an AutomationCondition that is true if the target's freshness is PASS."""
+        from dagster._core.definitions.declarative_automation.operands import (
+            FreshnessResultCondition,
+        )
+
+        return FreshnessResultCondition(state=FreshnessState.PASS)
```

Comment:
for consistency with `check_result` / the general pattern of `AutomationCondition.x` -> a label of `x`, we should probably update the static constructors to be:

```python
return FreshnessResultCondition(...).with_label("freshness_failed")
```